Dockerfile.geoip

```diff
@@ -0,0 +1,18 @@
+# Build
```

Comment:
Can you remove this file? One `Dockerfile` is enough, and the Maxmind guys are very picky about redistribution/attribution of the database files.